### DISCUSSION TITLE
Critical dependency: the request of a dependency is an expression

### DISCUSSION THREAD

**Discussion by TLovers** [PrismLibrary/Prism#2524] [General]
当我在nuxt 中使用prismjs 启动项目的时候命令行总是报警告 Critical dependency: the request of a dependency is an expression 。页面会报错 Cannot find module './prism-markup'
我查看prismjs/component/index.js 好像是 45行 require(pathToLanguage)的问题
当我尝试require(`${pathToLanguage}`) 就变得正常
我想知道这个是不是Bug

**Comment by dansiegel** [comment]
@czloveyou this is not the repo for PrismJS

**Reply by TLovers** [reply]
Oh, wrong excuse me 😄
